test/runtests.jl

```diff
@@ -101,7 +161,27 @@ end
 
 println("Testing input file 4:")
 @testset "Test input file 4" begin
-    outputData = OutputData(INPUT_TEST_PATHS[4])
+    outputData = 0
+    try
+        outputData = OutputData(INPUT_TEST_PATHS[4])
+    catch e
+        id = 0
+        try 
+            id = e.id
+        catch err 
+            println("Unexpected Error! Skipping Tests!")
+            return
+        end
+        if id == Int(InputParser.ParsingErrorId)
+            println("Error! Skipping Tests!")
+        elseif id == Int(InputParser.SoilNumberErrorId)
+            println("Error: Invalid input file!")
+            println("\t>Line $(e.line): Soil layer number must be larger than previous one!") 
```

Comment:
I see essentially the same code over and over again in this change.  Can you refactor the code so that you only have this once?